dlc-manager/src/manager.rs

```diff
@@ -924,6 +926,19 @@ where
         Ok(msg)
     }
 
+    /// Reject a channel that was offered. Returns the [`dlc_messages::channel::Reject`]
+    /// message to be sent as well as the public key of the offering node.
+    pub fn reject_channel(&self, channel_id: &ChannelId) -> Result<(Reject, PublicKey), Error> {
```

Comment:
If this should be called only by the accept party, it feels like it would be good to have a check for that.

Reply:
300cbf1dd884f0fa8dcbfb6356604699880f06e0